src/authorization/AuthorizationsDefinition.ts

```diff
@@ -102,6 +102,10 @@ export const AUTHORIZATION_DEFINITION: AuthorizationDefinition = {
   },
   admin: {
     grants: [
+      {
+        resource: Entity.TENANT, action: [Action.READ, Action.UPDATE],
+        attributes: ['id', 'name', 'email', 'logo', 'address']
```

Comment:
You are missing some attributes for both READ and UPDATE actions.
 Since both actions also require different attributes would be best to separate them.

Reply:
I'v separated the actions